docs/architecture/porting-existing-aspnet-apps/logging-differences.md

```diff
@@ -1,54 +0,0 @@
----
```

Comment:
I don't see a redirection for this one. Is that intentional?

Reply:
@ardalis @IEvangelist This file wasn't added to the TOC when it was added on Feb 11 in https://github.com/dotnet/docs/pull/22167. Should it be deleted or added to the TOC?

Reply:
Yes, I believe that should be added to the TOC, thanks for catching that!